src/lib/logger.ts

```diff
@@ -154,6 +158,7 @@ export class Logger {
     const logMessage: LogMessage = {
       application: appName,
       activity: activityUrl,
+      offeringId,
```

Comment:
I'm never sure if we should try to use resourceLinkId here or just stick with offeringId.
resourceLinkId is what it would be called in an LTI system, offeringId is what we call it in our portal.

At some point we hope to launch CLUE and the Activity Player from a LTI LMS in addition to launching it from our portal. That doesn't mean we have to log the id with a specific name. It just means that when we get the id from these LMS it will have that name.

It is fine to just leave this as offeringId, I just wanted you to be aware of this conundrum. Probably we use offeringId in most places that are visible to researchers.